cmd/logcli/query.go

```diff
@@ -72,8 +73,8 @@ func doQuery() {
 	i = iter.NewQueryResponseIterator(resp, d)
 
 	for i.Next() {
-		ls := labelsCache(i.Labels())
-		ls = subtract(ls, common)
+		fullls := labelsCache(i.Labels())
```

Comment:
Was there any specific reason for this change?

Reply:
In `default` mode, we print common labels at the top, and only specific labels at the log line. The `jsonl` mode is meant for parsing by scripts, etc. and it is hard to parse a common labels line diverting from the usual scheme. So I thought it would be simpler to emit all labels at any time